test/parallel/parallel_test_runner.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Get current test file name
+TESTFILE_NAME=${PGAPPNAME##pg_regress/}
+# Set different name for each test database
+# As pg_regress does not support cleaning db after each test
+TEST_CASE_DB="ldb_parallel"
+# Set database user
+if [ -z $DB_USER ]
+then
+     echo "ERROR: DB_USER environment variable is not set before test_runner.sh is run by pg_regress"
+     exit 1
+fi
+
+# Drop db after each test on exit signal
+function drop_db {
+  cat <<EOF | psql "$@" -U ${DB_USER} -d postgres -v ECHO=none >/dev/null 2>&1
+    SET client_min_messages=ERROR;
+    DROP DATABASE "${TEST_CASE_DB}";
+EOF
+}
+
+if [[ "$TESTFILE_NAME" =~ ^end ]]; then
```

Comment:
can we document somewhere. that `begin.sql` and `end.sql` are treated differently in the runner? We can just add comments in this file

Reply:
sounds good, I'll add comments in the schedule and the files themselves too